### DISCUSSION TITLE
[archived] Somebody could share a sample C3D of a nice-looking task (gymnastics/acrobatics/etc.)?

### DISCUSSION THREAD

**Discussion by felixchenier** [kineticstoolkit/kineticstoolkit#90] [2. Development: Ideas, new features, doc, etc.]
Hello everyone

I want to enhance the "Getting started" section of Kinetics Toolkit to include a very short starter code that would demonstrate some main features in a glance, so that one has a nice minimal sample code to start. This would be showcased in the very first sections of the webpage, and I want it to be appealing.

In my lab, I only have C3Ds for wheelchair propulsion, which is nice but not "cool" (it doesn't really have a WOW effect). Would someone have a short c3d of a full body acquisition during a cool-looking task, that would be free of rights, that could be included in this showcase? NB Please be sure to have the rights (ethics, etc.) to share such data.

If so, you can share it here.

Thank you!

**Comment by modenaxe** [comment]
only gait, running, stair negotiation and chair rising. Happy to share though!

**Reply by felixchenier** [reply]
Thank you for the offer @modenaxe. I'll wait a few weeks (don't have time for it anyway at the moment) and I'll come back to you if needed.

**Comment by felixchenier** [comment]
I got a very nice sample of tennis serve from Fraje Watson of University College London. Thanks Fraje!
